### PR TITLE
BHoM_UI: New boolean toggle component

### PR DESCRIPTION
**Pull request by albinber**
### Issues addressed by this PR

Closes #

### CODE REVIEW

File: BHoM_UI/Components/UI/FalseStartToggle.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        public override System.Drawing.Bitmap Icon_24x24 { get; protected set; } = Properties.Resources.BooleanToggle;
+
+        public override Guid Id { get; protected set; } = new Guid("e1e0d75f-41b0-4268-ae5c-a55673d77851");
+
+        public override string Category { get; protected set; } = "UI";
+
+        public override string Name { get; protected set; } = "FalseStart";
+
+        public override string Description { get; protected set; } = "A boolean toggle that changes to false on load.";
```
Comment (FraserGreenroyd) [inline BHoM_UI/Components/UI/FalseStartToggle.cs:53]:
```suggestion
        public override string Description { get; protected set; } = "A boolean toggle that defaults to false on load.";
```

File: BHoM_UI/Components/UI/FalseStartToggle.cs [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        /**** Properties                  ****/
+        /*************************************/
+
+        public override System.Drawing.Bitmap Icon_24x24 { get; protected set; } = Properties.Resources.BooleanToggle;
+
+        public override Guid Id { get; protected set; } = new Guid("e1e0d75f-41b0-4268-ae5c-a55673d77851");
+
+        public override string Category { get; protected set; } = "UI";
+
+        public override string Name { get; protected set; } = "FalseStart";
```
Comment (FraserGreenroyd) [inline BHoM_UI/Components/UI/FalseStartToggle.cs:51]:
```suggestion
        public override string Name { get; protected set; } = "FalseStartToggle";
```

File: BHoM_UI/Components/UI/FalseStartToggle.cs [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+        /*************************************/
+        /**** Constructors                ****/
+        /*************************************/
+
+        public FalseStartToggleCaller() : base()
+        {
+            SelectedItem = false;
+            Description = "Boolean toggle";
+            Name = "";
```
Comment (FraserGreenroyd) [inline BHoM_UI/Components/UI/FalseStartToggle.cs:76]:
```suggestion
```

File: BHoM_UI/Components/UI/FalseStartToggle.cs [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            OutputParams = new List<ParamInfo>() { new ParamInfo { DataType = typeof(bool), Kind = ParamKind.Output, Name = Name, Description = Description, IsRequired = true } };
+        }
+
+
+
```
Comment (FraserGreenroyd) [inline BHoM_UI/Components/UI/FalseStartToggle.cs:81]:
```suggestion
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer
@BHoMBot check core

**Review by FraserGreenroyd — APPROVED** [review]
image

This looks great @albinber , great work and great team-work/collaboration on this.

I've reviewed the code and that looks fine.

I've tested in Grasshopper by setting a few components on the canvas, saving, and reopening the script and the components load in defaulting to `false`.

Happy to deploy to alphas.

**Review by Tom-Kingstone — APPROVED** [review]
image
Tested on loading, and any toggles that were true are set to false, good job @albinber

**Pull request merged**